documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,210 @@
+- As a **prospective applicant**, I want:
+  - To be able to be able to search with filters applied, so that the only search results returned are ones that match both the keyword and field-based criteria I specified
+  - The search results to have a link to the opportunity on grants.gov, so that I don't have to repeat my search on grants.gov after finding an opportunity I want to apply for on simpler.grants.gov
+  - To receive an incentive if I participate in user research for simpler.grants.gov, so that I am fairly compensated for the time that I invest in testing out new features and answering questions
+  - Filters and search terms to be added to the url, so that I can save or share the link to a particular set of search results
+- As a **project maintainer**, I want:
+  - To be notified when simpler.grants.gov experiences downtime, so that I can quickly troubleshoot the issue with minimal impact on end users
+  - The system architecture we create for this deliverable to support future search features as well, so that we don't need to discard and re-build critical infrastructure
+
+## Technical description
+
+### Search API
+<!-- Optional -->
+
+The ability to search and filter opportunities should be provided through an API endpoint, which allows the user to:
+
+- Search the list of opportunities with open ended key words
+- Filter the search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- Sort the search results based on the value of *at least*
+
+Through the design and development of this API endpoint, the team should answer the following questions and record the decisions in one or more ADRs:
+
+- How should search and filter criteria be passed to the API endpoint used for search?
+- How should we support basic logic such as greater than, less than, as well as and/or/not conditions?
+- Which fields should support filtering and sorting?
+- Do we use an existing search service (e.g. Elasticsearch) or build our own?
+
+### Search Page
+
+The primary page we'll need to deliver for the user interface (UI) is a search page, which should allow users to:
+
+- Enter key words to search the list of funding opportunities
+- Filter search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- View a common set of fields for each opportunity returned by the search (e.g. title, award date, summary, etc.)
+- Sort the set of results returned by *at least* one structured field (e.g. award date, title, etc.)
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Simpler site translations are live in PROD (if necessary). Translations of the search results content coming from grants.gov will not be included in this 30k ft deliverable. 
```

Comment:
what does "if necessary" mean here?

Reply:
That might have been hold over language from the template. I took it out from this bullet point.